utilities/test_suite/HOST/testAllScript.sh

```diff
@@ -56,11 +59,16 @@ if (( "$#" < 3 )); then
     DECODER_TYPE="0"
     NUM_RUNS="1"
     PRESERVE_OUTPUT="1"
-    BATCH_SIZE="1"
     CASE_LIST=()
     for ((case="$CASE_MIN";case<="$CASE_MAX";case++))
     do
         CASE_LIST+=("$case")
+        if [ "$case" -eq "82" ]
+        then
+            BATCH_SIZE="3"
```

Comment:
@sampath1117 Please confirm here if changing batch size for a specific case has any other repercussions on QA/unit/perf tests.